package.json

```diff
@@ -222,16 +222,23 @@
   },
   "dependencies": {
     "adal-node": "0.1.28",
-    "azure-arm-resource": "7.0.1",
+    "copy-paste": "^1.3.0",
+    "ms-rest-azure": "2.5.9",
+    "vscode-nls": "4.0.0",
+    "opn": "^5.1.0",
```

Comment:
Looks like you added some extra dependencies on accident. These are unrelated to `@azure/` packages.